xrpl/asyncio/transaction/main.py

```diff
@@ -269,18 +269,16 @@ async def _check_fee(transaction: Transaction, client: Optional[Client] = None)
     Raises:
         XRPLException: if the transaction fee is higher than the expected fee.
     """
-    # Calculate the expected fee from the network load and transaction type
-    expected_fee = await _calculate_fee_per_transaction_type(transaction, client)
+    expected_fee = max(
+        xrp_to_drops(0.1),  # a fee that is obviously too high
+        await _calculate_fee_per_transaction_type(transaction, client),
+    )
 
     if transaction.fee and int(transaction.fee) > int(expected_fee):
         raise XRPLException(
-            "Fee value: "
-            + str(drops_to_xrp(transaction.fee))
-            + " XRP exceeds the "
-            + str(drops_to_xrp(expected_fee))
-            + " maximum XRP fee limit for "
-            + transaction.transaction_type
-            + " transaction"
+            f"Fee value: {str(drops_to_xrp(transaction.fee))} XRP is likely entered "
+            "incorrectly, since it is much larger than the "
+            f"{str(drops_to_xrp(expected_fee))} maximum XRP fee limit."
```

Comment:
Saying it's "much larger than the maximum XRP fee limit" feels wrong here (especially since "the" is referring to a configurable setting in the client). 

```suggestion
            f"{str(drops_to_xrp(expected_fee))} typical XRP transaction cost."
```

The error message _could_ maybe also say something along the lines of, "If you meant to burn this much XRP, change (some setting) to bypass this check."